src/externalServices/storeTrigger/interfaces.ts

```diff
@@ -8,7 +8,16 @@ export interface StoreTriggerResponse {
 
 export interface StoreTriggerConfig {
   url: string;
-  subUrl: string;
+  subUrl: {
```

Comment:
Remove the suburls. We will use it hard-coded since we have more than one suburl. There is no added value to do this configurabilly